zerver/lib/queue.py

```diff
@@ -3,7 +3,7 @@
 from django.conf import settings
 import pika
 from pika.adapters.blocking_connection import BlockingChannel
-from pika.spec import Deliver, BasicProperties
+from pika.spec import Basic, BasicProperties
```

Comment:
I think you have consistently used `pika.BasicProperties` in your code. So you can probably change it to `from pika.spec import Basic`.